packages/vuepress-theme-vt/layouts/Layout.vue

```diff
@@ -92,16 +97,32 @@ export default {
       )
     },
 
-    shouldShowSidebar () {
+    showShowSideSection() {
       const { frontmatter } = this.$page
       return (
         !frontmatter.home
         && !frontmatter.api
+        && !frontmatter.pageLayout
+      )
+    },
+
+    shouldShowSidebar() {
+      const { frontmatter } = this.$page
+      return (
+        this.showShowSideSection
         && frontmatter.sidebar !== false
         && this.sidebarItems.length
       )
     },
 
+    shouldShowToc() {
+      const { frontmatter } = this.$page
+      return (
+        this.showShowSideSection
+        && frontmatter.toc !== false
```

Comment:
It looks like add new frontmatter variables `pageLayout ` and `toc`.  
Does it need to add into `packages/vuepress-docs/docs/guide/frontmatter.md`?

Reply:
It has been documented but not implemented:

https://vuepress-theme-vt.vercel.app/api/config-frontmatter.html#toc